docs/units/equivalencies.rst

```diff
@@ -273,53 +323,78 @@ with the `~astropy.units.equivalencies.beam_angular_area` equivalency::
-deals with beam input/output and various operations more directly.
+Note that the `radio_beam <https://github.com/radio-astro-tools/radio-beam>`_
+package deals with beam input/output and various operations more directly.
+
+.. EXAMPLE END
 
 Temperature Energy Equivalency
 ------------------------------
 
 This equivalency allows conversion between temperature and its equivalent
 in energy (i.e., the temperature multiplied by the Boltzmann constant),
 usually expressed in electronvolts. This is used frequently for
-observations at high-energy, be it for solar or X-ray astronomy. Example::
+observations at high-energy, be it for solar or X-ray astronomy.
+
+Example
+^^^^^^^
+
+.. EXAMPLE START: Temperature Energy Equivalency
+
+To convert between temperature and its equivalent in energy::
 
     >>> import astropy.units as u
     >>> t_k = 1e6 * u.K
     >>> t_k.to(u.eV, equivalencies=u.temperature_energy())  # doctest: +FLOAT_CMP
     <Quantity 86.17332384960955 eV>
 
+.. EXAMPLE END
+
 .. _tcmb-equivalency:
 
 Thermodynamic Temperature Equivalency
 -------------------------------------
 
 This :func:`~astropy.units.equivalencies.thermodynamic_temperature`
-equivalency allows conversion between Jy/beam and "thermodynamic
-temperature", :math:`T_{CMB}`, in Kelvins. Example::
+equivalency allows conversion between ``Jy/beam`` and "thermodynamic
+temperature", :math:`T_{CMB}`, in Kelvins.
+
+Examples
+^^^^^^^^
+
+.. EXAMPLE START: Thermodynamic Temperature Equivalency
+
+To convert between ``Jy/beam`` and thermodynamic temperature::
 
     >>> import astropy.units as u
     >>> nu = 143 * u.GHz
     >>> t_k = 0.002632051878 * u.K
     >>> t_k.to(u.MJy / u.sr, equivalencies=u.thermodynamic_temperature(nu))  # doctest: +FLOAT_CMP
     <Quantity 1. MJy / sr>
 
-By default, this will use the :math:`T_{CMB}` value for the 'default cosmology'
-in astropy, but it is possible to specify a custom :math:`T_{CMB}` value for a
+By default, this will use the :math:`T_{CMB}` value for the default cosmology in
+``astropy``, but it is possible to specify a custom :math:`T_{CMB}` value for a
 specific cosmology as the second argument to the equivalency::
 
     >>> from astropy.cosmology import WMAP9
     >>> t_k.to(u.MJy / u.sr, equivalencies=u.thermodynamic_temperature(nu, T_cmb=WMAP9.Tcmb0))  # doctest: +FLOAT_CMP
     <Quantity 0.99982392 MJy / sr>
 
+.. EXAMPLE END
+
 Molar Mass AMU Equivalency
 --------------------------
 
-This equivalency allows conversion
-between the atomic mass unit and the equivalent g/mol.
-For reference to why this was added,
-refer to `astropy GitHub issue 6040 <https://github.com/astropy/astropy/issues/6040>`_
-The following is an example of it's usage:
+This equivalency allows conversion between the atomic mass unit and the
+equivalent g/mol.  For reference as to why this was added, refer to `astropy
+GitHub issue 6040 <https://github.com/astropy/astropy/issues/6040>`_.
```

Comment:
Again, not your issue, but it seems crazy to allow the text to refer to some github issue about why it was introduced. Maybe instead refer to https://www.nist.gov/si-redefinition/definitions-si-base-units? (With some text... I may be able to help but will finish review first)